apps/website/src/routes/docs/headless/progress/examples/hero.tsx

```diff
@@ -0,0 +1,28 @@
+import { component$, useSignal, useVisibleTask$ } from '@builder.io/qwik';
+import * as Progress from '@qwik-ui/headless';
+
+export default component$(() => {
+  const progress = useSignal(30);
+
+  useVisibleTask$(() => {
+    setTimeout(() => {
+      progress.value = 50;
+    }, 1000);
+  });
+
+  return (
+    <>
+      <Progress.Root
```

Comment:
Recently, I have been using regular css for the headless examples, almost like a "wireframe", to visually show the building blocks. Check out select.css in the select examples.

I would take a look at how we do it in the select. It also makes the code examples much cleaner.

At some point in the future, we might have a dropdown with different styles, but the headless should showcase the unstyled pieces. This is something we need to refactor in the other headless components.

Reply:
Makes sense, I replicated the select approach by moving the CSS to a specific file.